app/routes/session.py

```diff
@@ -178,7 +182,19 @@ def get_sign_out():
 
 @session_blueprint.route("/signed-out", methods=["GET"])
 def get_signed_out():
-    return render_template(template="signed-out")
+    if not cookie_session:
+        return redirect(url_for("session.get_session_expired"))
+
+    survey_type = get_survey_type()
+    survey_config = get_survey_config(theme=survey_type)
```

Comment:
Minor, don't need to fetch survey type anymore. It will behind the scenes.
```suggestion
    survey_config = get_survey_config()
```